### ISSUE TITLE
Update OpenGrid Video

### ISSUE THREAD

**Issue by PriyaDoIT** [Chicago/opengrid#263]

**Comment by tomschenkjr** [comment]
@PriyaDoIT - per our conversation today, [here is a script](https://github.com/Chicago/opengrid/files/952507/example_tutorial_script.txt) I used to create an [open data portal](https://www.youtube.com/watch?v=yjijSAL_6EU&list=PLRUxrvb7sFARx_WfL3ZTjM6DmhvZl34Mm) tutorial. 

Don't worry about the formatting as much (I use a lot of spacing since it's easier to read and then narrate).

The subtasks basically fall into these steps:

- [ ] Create script(s) for the video(s)
- [ ] Record the audio narration for the script
- [ ] Record the video capture
- [ ] Edit the video and audio narration into a final file
- [ ] Upload the video to the [OpenGrid playlist](https://www.youtube.com/watch?v=pzhmbtf2Vp8&list=PLRUxrvb7sFAQ6-huyxTGFuG6lzpxeDU8J)

**Comment by PriyaDoIT** [comment]
Hi @tomschenkjr , I thought that you would record the video and write the script?  I started some of it here:
https://chicagogov.sharepoint.com/sites/doit/datascience/advancedanalytics/_layouts/15/WopiFrame.aspx?sourcedoc=%7BF1E2342C-7AD7-48EA-BBBB-AFB269DB9DDE%7D&file=Script.docx&action=default

let me know if you like this format if you want something else before I continue.

**Comment by tomschenkjr** [comment]
Yeah, this works. I'll record the audio, screen capture and will edit them.

_
Tom Schenk Jr.
Chief Data Officer
Department of Innovation and Technology
City of Chicago
(312) 744-2770
tom.schenk@cityofchicago.org | @ChicagoCDO
data.cityofchicago.org | opengrid.io | digital.cityofchicago.org | chicago.github.io | dev.cityofchicago.org


________________________________
